diff --git a/node_modules/@react-native-firebase/storage/ios/RNFBStorage/RNFBStorageCommon.h b/node_modules/@react-native-firebase/storage/ios/RNFBStorage/RNFBStorageCommon.h
index 36f601d..9992250 100644
--- a/node_modules/@react-native-firebase/storage/ios/RNFBStorage/RNFBStorageCommon.h
+++ b/node_modules/@react-native-firebase/storage/ios/RNFBStorage/RNFBStorageCommon.h
@@ -16,6 +16,8 @@
  *
  */
 #import <Photos/Photos.h>
+#import <React/RCTBridgeModule.h>
+#import <FirebaseStorage/FirebaseStorage-Swift.h>
 
 @interface RNFBStorageCommon : NSObject
 
diff --git a/node_modules/@react-native-firebase/storage/ios/RNFBStorage/RNFBStorageCommon.m b/node_modules/@react-native-firebase/storage/ios/RNFBStorage/RNFBStorageCommon.m
index 0c90d3c..33994cf 100644
--- a/node_modules/@react-native-firebase/storage/ios/RNFBStorage/RNFBStorageCommon.m
+++ b/node_modules/@react-native-firebase/storage/ios/RNFBStorage/RNFBStorageCommon.m
@@ -17,6 +17,8 @@
  */
 
 #import <Firebase/Firebase.h>
+#import <React/RCTBridgeModule.h>
+#import <FirebaseStorage/FirebaseStorage-Swift.h>
 #import <MobileCoreServices/MobileCoreServices.h>
 
 #import "RNFBSharedUtils.h"
diff --git a/node_modules/@react-native-firebase/storage/ios/RNFBStorage/RNFBStorageModule.m b/node_modules/@react-native-firebase/storage/ios/RNFBStorage/RNFBStorageModule.m
index a3695b6..1ec9632 100644
--- a/node_modules/@react-native-firebase/storage/ios/RNFBStorage/RNFBStorageModule.m
+++ b/node_modules/@react-native-firebase/storage/ios/RNFBStorage/RNFBStorageModule.m
@@ -16,6 +16,8 @@
  */
 
 #import <Firebase/Firebase.h>
+#import <React/RCTBridgeModule.h>
+#import <FirebaseStorage/FirebaseStorage-Swift.h>
 #import <React/RCTUtils.h>
 
 #import "RNFBRCTEventEmitter.h"
